medusa-ui/src/main/resources/websocket.js

```diff
@@ -272,4 +272,22 @@ _M.sendEvent = function(originElem, e) {
     _M.ws.send(_M.injectVariablesIntoExpression(e));
 };
 
+_M.onEnter = function(element) {
+    element.addEventListener("keyup", function(event) {
+        if (event.key === "Enter") {
+            event.preventDefault();
+            let call = event.target.getAttribute("m-onenter");
+            _M.sendEvent(event.target, call);
+            return false;
+        }
+    });
+    element.addEventListener("keydown", function(event) {
+        if (event.key === "Enter") {
+            event.preventDefault();
+            return false;
+        }
+    });
+};
+
 _M.retryConnection();
+document.querySelectorAll("[m-onenter]").forEach(element => _M.onEnter(element));
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/medusa-ui/medusa/pullRequest?prid=7855407)
```suggestion
document.querySelectorAll("[m-onenter]").forEach((element) => _M.onEnter(element));
```